controllers/volumereplication_controller.go

```diff
@@ -201,14 +203,22 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{Requeue: true}, nil
 	}
 
-	instance.Status.LastCompletionTime = getCurrentTime()
+	var msg string
 	if instance.Spec.ReplicationState == replicationv1alpha1.Resync {
-		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked for resyncing")
+		msg = "volume is marked for resyncing"
 	} else {
-		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked "+string(instance.Spec.ReplicationState))
+		msg = "volume is marked " + string(instance.Spec.ReplicationState)
 	}
 
-	return ctrl.Result{}, err
+	instance.Status.LastCompletionTime = getCurrentTime()
+	err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, msg)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	r.Log.Info(msg, "Name", instance.Name)
```

Comment:
This can cause panic?

Reply:
No it should not. Why do you think so

Reply:
Looks good, i was thinking something like https://github.com/kube-storage/volume-replication-operator/pull/51 can happen.

Reply:
If you are referring this https://github.com/kube-storage/volume-replication-operator/pull/51 then you can see it was panic when we had 2 args and here we have 3